pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -872,6 +872,7 @@ https://github.com/catppuccin/vim/,HEAD,catppuccin-vim
 https://github.com/inkarkat/vim-AdvancedSorters/,,vim-advanced-sorters
 https://github.com/Konfekt/vim-CtrlXA/,,
 https://github.com/konfekt/vim-DetectSpellLang/,,
+https://github.com/fadein/vim-FIGlet/,HEAD,
```

Comment:
```suggestion
https://github.com/fadein/vim-figlet/,HEAD,
```

Reply:
The plugin is named with a capital 'FIG'. Is there a reason we don't want to maintain capitalization like we do for other plugins?

Reply:
The convention is for attribute names to be in (lower) kebab case, and GitHub URLs are case-insensitive. Some of the other plugins might have been added before the convention was formed.